http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/MockSamDAO.scala

```diff
@@ -65,6 +70,8 @@ class MockSamDAO extends SamDAO[IO] {
         IO.pure(projectOwners.get(authHeader).map(_.toList).getOrElse(List.empty)).map(_.asInstanceOf[List[A]])
       case SamResourceType.PersistentDisk =>
         IO.pure(diskCreators.get(authHeader).map(_.toList).getOrElse(List.empty)).map(_.asInstanceOf[List[A]])
+      case SamResourceType.App =>
+        IO.pure(List[A]()) //TODO
```

Comment:
can you add what some words indicating what the TODO is? simply something like "fix me" will do compared to nothing